collector-bitmovin-player/src/main/java/com/bitmovin/analytics/bitmovin/player/BitmovinSdkAdapter.java

```diff
@@ -149,10 +152,7 @@ private void removePlayerListener() {
     }
 
     @Override
-    public EventData createEventData() {
-        EventData data = factory.build(stateMachine.getImpressionId());
-
-        data.setAnalyticsVersion(BuildConfig.VERSION_NAME);
```

Comment:
Please test this in detail: Before it was set in both the `EventData` class and here. Maybe this has something to do with gradle package imports. (I guess it shouldn't be a problem as we can't have Core 1.2.0 with BitmovinCollector 1.3.0 etc., but please check the outgoing payload if everything works correctly.

Reply:
When changing the value in `build.gradle (Project: collector-android)` it will set this as the `analyticsVersion` in the payload. Tested locally with bitmovin and exoplayer example.